src/template/run_multi.rs

```diff
@@ -0,0 +1,255 @@
+use std::{collections::HashSet, io};
+
+use crate::template::{Day, ANSI_BOLD, ANSI_ITALIC, ANSI_RESET};
+
+use super::{
+    all_days,
+    timings::{Timing, Timings},
+};
+
+pub fn run_multi(days_to_run: HashSet<Day>, is_release: bool, is_timed: bool) -> Option<Timings> {
+    let mut timings: Vec<Timing> = vec![];
```

Comment:
```suggestion
    let mut timings: Vec<Timing> = Vec::with_capacity(days_to_run.len());
```

To avoid re-allocation.